client/my-sites/upgrades/checkout/domain-details-form.jsx

```diff
@@ -160,11 +198,32 @@ class DomainDetailsForm extends Component {
 		} );
 	}
 
+	getAllFieldValues = () => {
+		const allFieldValues = Object.assign(
+			{},
+			formState.getAllFieldValues( this.state.form ),
+			this.state.registrantExtraInfo
```

Comment:
Yes, this is how I decided to go for now, but we won't know if it's right until we get validation and pre-populating-from-the-db working.

I went this way now because:
- What do we do with validation errors for fields that aren't visible?
- There's no meaningful notion of "complete" with extra info anyway, what we care about depends on the domains and on the fields themselves
- We're going to need to do some reasoning on the back end to figure out which fields matter either way
- Backend processing is more plentiful than bandwidth
- There are no dependencies yet between the "core" data and the "extra" data.
- the empty fields will become meaningful when we repopulate the form from the database.

Redux gives us a solid answer to "where do we put the errors" (they go in the state!), so I think that what we want to do is send everything we've got from the user, but not pad it out with empty fields.